components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -249,12 +251,37 @@ message WorkspaceInstanceStatus {
     // Admission describes who can access a workspace instance and its ports.
     AdmissionLevel admission = 6;
 
+    // ports is the list of served ports in the workspace.
```

Comment:
Exposed, not served. It is only about access regardless of whether anyone is listening